lib/index.js

```diff
@@ -41,6 +41,22 @@ bedrock.events.on('bedrock-mongodb.ready', function init(callback) {
 });
 
 /**
+ * Check for the existence of an identity.
+ *
+ * @param id the ID of the identity to check.
+ * @param callback(err, exists) called once the operation completes.
+ */
+api.exists = function(id, callback) {
+  database.collections.identity.find({
```

Comment:
Yup, saw it. We'll probably use this `exists` API. I just want to make sure it's consistent with our other ones.